tests/clickhouse-test

```diff
@@ -116,7 +116,7 @@ def clickhouse_execute_http(base_args, query, timeout=30, settings=None, default
 def clickhouse_execute(base_args, query, timeout=30, settings=None):
     return clickhouse_execute_http(base_args, query, timeout, settings).strip()
 
-def clickhouse_execute_json(base_args, query, timeout=30, settings=None):
+def clickhouse_execute_json(base_args, query, timeout=60, settings=None):
```

Comment:
@azat  why this can take more than 30 seconds ?

Reply:
I guess due to parallel tests, for example [here](https://s3.amazonaws.com/clickhouse-test-reports/32928/ddd5bebe555ce8feebcdd339e47fc45184c20dd1/stateless_tests__thread__actions__[1/3].html), you can also find details in the commit message - https://github.com/ClickHouse/ClickHouse/pull/35745/commits/7d140c9a59c1760d7f89d6b1cbc3ef149f3324e5